### PR TITLE
Statsenko

### PR DESCRIPTION
**Pull request by VStako**
add enrollment

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 def pytest_addoption(parser):
     parser.addoption("--browser", action="store", default="firefox")
-    parser.addoption("--base_url", action="store", default="http://localhost:9000/")
-
+    #parser.addoption("--base_url", action="store", default="http://localhost:9000/")
+    parser.addoption("--base_url", action="store", default="http://194.44.198.221/#/signin")
```
Comment (ITsvetkoFF) [inline conftest.py]: leave it without #/signin please

File: pages/enrollments/add/enrollment_main_page.py [hunk 2]
```diff
@@ -1 +1,196 @@
...
+        self.checkbox_is_privilege.click()
+        self.radiobutton_getting_education.click()
+        self.radiobutton_is_interview.click()
+        self.checkbox_is_hostel.click()
+        self.search_offers_field.click()
+        self.find_element_in_ui_select(self.list_form_ui_select(), "Фізичний").click()
+        self.choose_form_of_education.click()
+        self.find_element_in_ui_select(self.list_form_ui_select(), "Бакалавр").click()
+        self.button_choose_specialties.click()
+        sleep(3)
```
Comment (ITsvetkoFF) [inline pages/enrollments/add/enrollment_main_page.py]: try to get rid of sleeps, pleeease!)

**Comment by VStako** [comment]
I corrected my test and you can merge my commit with master =)

File: pages/enrollments/add/enrollment_main_page.py [hunk 3]
```diff
@@ -1 +1,210 @@
...
+        "total_score": "77",
+        "priority": "3",
+        "structural_unit": "Фізичний",
+        "type_of_entry": "За результатами іспитів",
+        "detailing_start": "Учасник міжнародної олімпіади",
+        "date_closing": "2016-04-04"
+    }
+
+    OK_FOR_INPUT_FIELD = (By.CSS_SELECTOR, "div[class='input-group'] * button[class='btn btn-primary']")
+    SECOND_PERSON = (By.XPATH, "html/body/div[5]/div/div/div[2]/div[2]/table/tbody/tr[6]/td[2]")
```
Comment (ITsvetkoFF) [inline pages/enrollments/add/enrollment_main_page.py:27]: too loooong)

**Pull request merged**